src/index.html

```diff
@@ -1,17 +1,179 @@
+            for="acceptAgreement"
+            class="control-label"
+          >
+            I accept the term of the agreement
+            <input
+              type="checkbox"
+              name="acceptAgreement"
+              id="acceptAgreement"
+              value="no"
+              required
+            >
+          </label>
+        </fieldset>
+        <fieldset class="fieldset">
+          <legend>Registration</legend>
+          <label
+            for="email"
+            class="control-label"
+          >
+            E-mail:
+            <input
+              type="email"
+              name="email"
+              id="email"
+              placeholder="email@example.com"
+              required
+            >
+          </label>
+          <label
+            for="password"
+            class="control-label"
+          >
+            Password:
+            <input
+              type="password"
+              name="password"
+              id="password"
+              minlength="6"
+              maxlength="20"
+              required
+            >
+          </label>
+        </fieldset>
+        <fieldset class="fieldset">
+          <legend>An interesting fact about you!</legend>
+          <label class="control-label">
+            Do you love cats?
+            <label for="yes">
+              <input
+                id="yes"
+                type="radio"
+                name="cats"
+                value="yes"
+              >
+               Yes 
+            </label>
+            <label for="no">
+              <input
+                id="no"
+                type="radio"
+                name="cats"
+                value="no"
+              > No 
+            </label>
+          </label>
+          <label class="control-label">
+            What is your favorite color?
+            <input type="color">
+          </label>
+          <label class="control-label">
+            What time do you go to bed?
+            <input
+              type="time"
+              step="1"
+            >
+          </label>
+          <label class="control-label">
+            What are your favorite brand of cars?
+            <select name="favorite cars" multiple>
+              <option value="BMW">BMW</option>
+              <option value="Audi">Audi</option>
+              <option value="Lada">Lada</option>
+            </select>
+          </label>
+          <label class="control-label">
+            How do you rate our work?
+            <input
+              type="range"
+              min="0"
+              max="11"
+              value="0"
+            >
+          </label>
+        </fieldset>
+        <fieldset class="fieldset">
+          <legend>Additional info:</legend>
+          <label class="control-label">
+            Comments:
+            <textarea
+              name="comments"
+              cols="20"
+              rows="2"
+              autocomplete="off"
+            >
+            </textarea>
```

Comment:
```suggestion
            ></textarea>
```
to prevent this
<img width="309" alt="image" src="https://user-images.githubusercontent.com/53611910/216776820-8e1126db-46a2-4295-a99e-f55163b0c8b9.png">

Reply:
![image](https://user-images.githubusercontent.com/56869338/216778980-70c19e5d-3964-411a-a29b-4852a35e4a2e.png)
видає помилку через те. що не на одному рівні